templates/todo/projects/java-mongo-aca/.repo/bicep/infra/main.bicep

```diff
@@ -34,6 +38,9 @@ param apiImageName string = ''
 @description('The image name for the web service')
 param webImageName string = ''
 
+@description('The base URL used by the web service for sending API requests')
+param webApiBaseUrl string = ''
```

Comment:
it kind of confusing to read `webApiBaseUrl`.
Is it the web or the api url?
Wondering if we should just name it `apiBaseUrl`, like the parameter used in `web-container-app.bicep` ?

Reply:
I'm trying to avoid the confusion that this **does not** modify the URL that the API is running on. It is what the web service is looking at for making service API requests (it's a property of the `web` service). Similar to `REACT_APP_API_BASE_URL`, except I'm trying to avoid referencing `react` in the bicep.

Reply:
I'm not sure what you mean. Remote vs local has no effect on the naming.

`webX` <- a property that affects the web
`apiBaseUrl` <- the API base URL

`webApiBaseUrl` <- The URL that the web service uses to route requests

Reply:
I think there's some confusion because `webApi` itself could mean the "web API". Keeping it as-is though since it does follow the convention.